views/room.ejs

```diff
@@ -13,17 +13,12 @@
     <!--[if IE]><link rel="shortcut icon" href="/favicon.ico"><![endif]-->
     <link rel="icon" href="/favicon.png">
 
-    <link rel="stylesheet prefetch" href="/css/endMeeting.opentok.css">
-
     <link rel="stylesheet prefetch" href="/css/room.opentok.css">
-    <link rel="stylesheet prefetch" href="/css/buttons.css" />
-    <link rel="stylesheet prefetch" href="/css/landing.opentok.css">
 
     <link href="/css/fonts.css" rel="stylesheet">
     <link rel="chrome-webstore-item" href="https://chrome.google.com/webstore/detail/<%=chromeExtensionId%>">
     <link rel="stylesheet" rel="preload" href="https://assets.tokbox.com/solutions/css/style.css">
     <link rel="stylesheet" rel="prefetch" href="/css/annotation.opentok.css">
-    <link rel="stylesheet prefetch" href="/css/checkboxes.css" />
```

Comment:
We can delete the checkboxes.css file, I think. And the reference to it in .gitignore.

Reply:
updated in latest commit